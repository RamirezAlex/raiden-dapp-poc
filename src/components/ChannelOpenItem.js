import React from 'react'
const ChannelOpenItem = (props) => {
  const { channel, count } = props
  return (
    <tr>
      <td>{ count }</td>
      <td>{ channel.channelIdentifier }</td>
      <td>{ channel.participant1 }</td>
      <td>{ channel.participant2 }</td>
    </tr>
  )
}

export default ChannelOpenItem
