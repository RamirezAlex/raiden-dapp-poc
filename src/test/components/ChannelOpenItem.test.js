import React from 'react'
import ChannelOpenItem from '../../components/ChannelOpenItem'
import { create } from 'react-test-renderer'

describe('ChannelOpenItem', () => {
  it('Shoulde render a ChannelOpenItem', () => {
    expect(ChannelOpenItem).not.toBeNull()
    const channel = {
      channelIdentifier: 24,
      participant1: '0x0E809A051034723beE67871a5A4968aE22d36C5A',
      participant2: '0x504300C525CbE91Adb3FE0944Fe1f56f5162C75C'
    }
    const component = create(<ChannelOpenItem channel={channel} count={3} />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
