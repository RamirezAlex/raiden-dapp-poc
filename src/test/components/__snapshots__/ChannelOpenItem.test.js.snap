// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChannelOpenItem Shoulde render a ChannelOpenItem 1`] = `
<tr>
  <td>
    3
  </td>
  <td>
    24
  </td>
  <td>
    0x0E809A051034723beE67871a5A4968aE22d36C5A
  </td>
  <td>
    0x504300C525CbE91Adb3FE0944Fe1f56f5162C75C
  </td>
</tr>
`;
